src/build/__init__.py

```diff
@@ -21,9 +21,12 @@
 if sys.version_info < (3,):
     FileNotFoundError = IOError
     PermissionError = OSError
+else:
+    unicode = str
 
 
 ConfigSettings = Dict[str, Union[str, List[str]]]
+Path = Union[str, bytes, unicode]
```

Comment:
I don't think we should name this `Path` because clashes with `Path` from pathlib. IMHO no need to define this, you can just use `Union[bytes, Text]` though I don't think we should allow bytes on python 3.

Reply:
Please review on #149.